### DISCUSSION TITLE
easy install of the development branch

### DISCUSSION THREAD

**Discussion by LiangMD-BGI** [deepmodeling/deepmd-kit#1097] [Q&A]
Hello all,

There is a new feature which I want to test but it is only available in the development branch of deepmd.
I have installed the deepmd using conda and offline packages but failed to install it manually.
I want to know if there is an easy way to install a development branch (such as using conda or pip)?

Best,
Liang

**Comment by njzjz** [comment]
No. You have to [install from source](https://docs.deepmodeling.org/projects/deepmd/en/latest/install/install-from-source.html), or [build conda packages](https://docs.deepmodeling.org/projects/deepmd/en/latest/install/build-conda.html) by yourself.
